app/views/subnets/import.html.erb

```diff
@@ -1,19 +1,19 @@
 <% title _("Import IPv4 subnets") %>
 
-<h2><%= _("The following IPv4 subnets have been found") %></h2>
-
 <%= form_for "subnets[]", :url => create_multiple_subnets_path do |f| %>
   <% display_all = !minimal?(@subnets) %>
-  <div class="accordion" id='accordion1'>
+  <div class="panel-group" id='accordion1'>
   <% @subnets.each do |subnet| %>
-    <% id = 'subnet_fields_'+subnet.to_s.gsub('/','_').gsub('.','_') %>
-    <div class="accordion-group">
-      <div class="accordion-heading" >
-        <a class="accordion-toggle" data-toggle="collapse" data-parent='#accordion1' href="<%= '#'+id %>" data-original-title='review' rel='twipsy' > <%= subnet %> </a>
+    <% id = 'subnet_fields_' + subnet.network_address.gsub('/','_').gsub('.','_') %>
+    <div class="panel panel-default">
+      <div class="panel-heading" >
+        <a class="panel-title" data-toggle="collapse" data-parent='#accordion1' href="<%= '#'+id %>" data-original-title='review' rel='twipsy' >
```

Comment:
data-original-title should probably be removed here too, it just shows an untranslated tooltip saying "review" and isn't very helpful!